### PR TITLE
Setup pattern grid editor

### PR DESCRIPTION
**Pull request by misterinterrupt**
@katherinel hey, I got this feature branch goin on over here. its a pretty large overhaul.
I gutted the whole structure that I had going. No more menu, just a simple grid with click behavior.
The idea here is that one clicks a square and it creates a 'pattern' object if it doesn't exist. it also sets that pattern as 'selected'
the style changes for the 3 states: doesn't exist / exists / exists and selected
maybe you have some comments? 

It might be simpler to just explore this branch and check it out without looking at the diff, since this is more like starting over.. https://github.com/misterinterrupt/Seeq/tree/setup_patternGridEditor

### CODE REVIEW

**Comment by misterinterrupt** [comment]
I am going to merge this and keep working since comments on it can be done form the new master branch once merged

File: app/src/components/PatternGrid.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import classNames from 'classnames';
+
+import addSelectPattern from '../actions';
```
Comment (katherinel) [inline app/src/components/PatternGrid.jsx]: why is this being imported here..?

File: app/src/components/Row.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import Cell from './Cell';
+
+const Row = ({children}) => (
+  <div class='patternRow'>{children}</div>
```
Comment (katherinel) [inline app/src/components/Row.jsx]: `className`

File: app/src/containers/SelectablePatternGrid.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {connect} from 'react-redux';
+
+import PatternGrid from '../components/PatternGrid';
+import { addSelectPattern } from '../actions';
+
+const getPatternGridData = (patterns, editorData) => ({
```
Comment (katherinel) [inline app/src/containers/SelectablePatternGrid.jsx]: this feels like it doesn't need to be its own function

**Pull request merged**